components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/consent/SSOConsentServiceImpl.java

```diff
@@ -987,17 +988,29 @@ private Receipt getReceipt(AuthenticatedUser authenticatedUser, String receiptId
         Receipt currentReceipt;
         String subject = buildSubjectWithUserStoreDomain(authenticatedUser);
         try {
+            initializeTenantRegistry(authenticatedUser);
             startTenantFlowWithUser(subject, authenticatedUser.getTenantDomain());
             currentReceipt = getConsentManager().getReceipt(receiptId);
         } catch (ConsentManagementException e) {
             throw new SSOConsentServiceException("Consent Management Error",
-                                                 "Error while retrieving user consents.", e);
+                    "Error while retrieving user consents.", e);
+        } catch (IdentityException e) {
+            throw new SSOConsentServiceException("Consent Management Error",
+                    "Error while initializing registry for the tenant domain: " + authenticatedUser.getTenantDomain(), e);
```

Comment:
This line has been exceeded 120 characters

Reply:
Fixed in 06271985d0d22192422630521503f04db25e5eff.